.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - dev
+  pull_request:
+    branches:
+      - master
```

Comment:
Wouldn't we want to run tests on all PR branches, not just the mainline ones?

Reply:
Hm yeah maybe just all `pull_request` and `push` events?